docs/upctl_server_storage_detach.md

```diff
@@ -0,0 +1,30 @@
+## upctl server storage detach
+
+Detaches a storage resource from a server
+
+```
+upctl server storage detach <UUID/Title/Hostname...> [flags]
+```
+
+### Options
+
+```
+      --address string   Detach the storage attached to this address.
```

Comment:
this would be required, then? should we document the fact that this (and any other similar) flag is required?